frontend/src/metabase/dashboard/components/DashCard.jsx

```diff
@@ -79,13 +81,13 @@ export default class DashCard extends Component {
             .map(card => ({
                 ...getIn(dashcardData, [dashcard.id, card.id]),
                 card: card,
-                duration: cardDurations[card.id]
+                isSlow: (slowCards[card.id] && card.query_average_duration && (card.query_average_duration > DATASET_USUALLY_FAST_THRESHOLD))
```

Comment:
I think this can just be `isSlow: slowCards[card.id]`. I feel like this is actually computing `isSlowAndUsuallyFast`,